src/api/get.js

```diff
@@ -0,0 +1,14 @@
+import Wreck from '@hapi/wreck'
+import { config } from '../config/config.js'
+import { createLogger } from '../common/helpers/logging/logger.js'
+
+const logger = createLogger()
+
+export async function get (url) {
+  try {
+    return await Wreck.get(`${config.get('backend.endpoint')}${config.get('backend.path')}${url}`)
+  } catch (error) {
+    logger.error(`Encountered error while calling the backend with url: ${config.backendEndpoint}${url}}`, error)
+    throw new Error()
```

Comment:
Thinking about this part.

I don't think we should throw an empty error as we'll lose the stack trace that helps us debug.

But the logger will log any error that isn't caught anyway. 

So is there any value in catching an error here other than saying it's a backend API call?

Will this extra log help?